src/linter/types.go

```diff
@@ -263,18 +263,12 @@ func (n typeNormalizer) NormalizeTypes(typeList []types.Type) {
 	}
 }
 
-func (n typeNormalizer) string2name(s string) *ir.Name {
```

Comment:
The function has been used once and such conversions 
are usually done in place, so there is no point in this function.